daemon/volumes.go

```diff
@@ -164,11 +177,25 @@ func (container *Container) registerVolumes() {
 		if rw, exists := container.VolumesRW[path]; exists {
 			writable = rw
 		}
+		runtime.LockOSThread()
+		defer resetLabeling()
+
+		if err := label.SetFileCreateLabel(container.MountLabel); err != nil {
+			logrus.Debugf("Unable to setup default labeling for volume creation %s: %v", path, err)
```

Comment:
I think it is at least `Warn`, but probably `Error` or returning error, because it is probably security issue.